prawns/governance/src/handlers/bounties.BountyProposed.event.ts

```diff
@@ -0,0 +1,42 @@
+import { EventHandlerContext, ExtrinsicHandlerContext } from '@subsquid/substrate-processor';
+import { decodeAddress } from '../utils';
+import { SubstrateBountyProposal, SubstrateNetwork } from '../model';
+import { getOrCreateGovernanceAccount } from '../utils';
+import { getBountiesBountyProposedEvent } from './typeGetters/getBountiesBountyProposedEvent';
+import { getBountiesProposedCall } from './typeGetters/getBountiesProposedCall';
+
+export default (network: SubstrateNetwork) =>
+  async (ctx: EventHandlerContext) => {
+    if (!ctx.event || !ctx.event.extrinsic) {
```

Comment:
It's just nitpicking but the event always exists here because you are tracking events.

It's only the extrinsic that cannot exist.

Reply:
I've seen this pattern in other event handlers in the governance pallet so thought I should copy it just to be safe. But yes it does make sense that the Squid runtime won't give us an empty event in an event handler 😁